main.js

```diff
@@ -24,3 +22,144 @@ form.addEventListener('submit', (event) => {
+    cssStyle: 'card-l',
+
+  },
+];
+
+const section = document.getElementById('portfolio');
+projects.forEach((project) => {
+  const div = document.createElement('div');
+  div.className = 'main-container';
+  div.innerHTML = ` <div class="${project.cssStyle} card"> 
+<div class="res-snapshoot">
+ <img class="Snapshoot" src= ${project.image}> 
+ </div>
+  <div class="res-frame">
+      <h1 class="title">${project.title}</h1> 
+      <ul class="frame">
+      <li class="detail-1">CANOPY</li>
+      <li class="detail">BACK END DEV</li>
+      <li class="detail">2015</li>
+      </ul>
+      <p class="detailp">${project.description}</p>
+      <div class="tags"> 
+      <button class="tag">html</button> 
+      <button class="tag">css</button>
+      <button class="tag">javascript</button>
+      </div>
+       <button type="button" value = ${projects.indexOf(project)} class="normal-button">${project.seeproject}</button> 
+     </div>
+</div>`;
+  section.appendChild(div);
+});
+
+const modal = document.getElementById('myModal');
+projects.forEach((project) => {
+  document.querySelectorAll('.normal-button').forEach((n) => n.addEventListener('click', () => {
+    modal.style.display = 'block';
+    modal.innerHTML = ` <div class="popup-card">
+<div class = "popup-close">
+        <h1 class = "popup-h1">${project.title}</h1>
+        
+</div>
+        <div class = "popup-frame">
+        <ul class="popup-frame">
+          <li class="detail-1">CANOPY</li>
+          <li class="detail">BACK END DEV</li>
+          <li class="detail">2015</li>
+        </ul>
+        </div>
+        <div class = "popup-img-card" style = "background-image: url('.${project.image}'); max-height: 300px; "></div>
+<div class = "popup-description">
+<div class = "desc">Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
+It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, </div>
+<div class = "pop-frame">  
+ <div class="tags">
+<button class="tag">html</button>
+<button class="tag">css</button>
+<button class="tag">javascript</button>
+</div>
+<div class="tags popup-tag">
+<button class="tag">github</button>
+<button class="tag">ruby</button>
+<button class="tag">Bootstraps</button>
+</div>
+<hr class = "popup-hr">
+<div class = "popup-link-button">
+<button class = "popup-button">See More <img src = "./image/pop.svg"></button>
+<button class = "popup-button">See More <img src = "./image/Icon-GitHub.svg"></button>
+</div>
+</div>
+      </div> `;
+  }));
+});
+
+window.onclick = function (event) {
+  if (event.target === modal) {
+    modal.style.display = 'none';
+  }
+};
```

Comment:
***[REQUIRED CHANGES]***

- Hey, great job on your modal pop-up window but there's an issue. A user can't find any button to close the modal pop-up. Please let's add a cancel button to help improve the user experience

Reply:
***[REQUIRED CHANGES]***

- Please, let's try and align our items to match the Figma design in the diagram below:
- Let's use the right font
- Let's use the right font-weight

**your design**

![align](https://user-images.githubusercontent.com/59397159/169664030-44f84e14-2e30-466e-a268-5184d32cc41d.png)


**Figma design**

![alignFigma](https://user-images.githubusercontent.com/59397159/169664071-08d40748-c716-4005-8e22-f9ef9af59453.png)